shared/src/data/default-state/patient-templates.ts

```diff
@@ -12,173 +13,1210 @@ const defaultMaleImage: ImageProperties = {
+        -calculateHealthChange(
+            healthPointsDefaults.redAverage,
+            healthPointsDefaults.blackMax,
+            2
+        ),
+        0
+    ),
+    [
+        {
+            matchingHealthStateId: noChangesState.id,
+            maximumHealth: healthPointsDefaults.blackMax,
+        },
+        {
+            matchingHealthStateId: noChangesState.id,
+            earliestTime: 3 * phaseTime,
+        },
+    ]
+);
+
+const redInstantTransportState = PatientHealthState.create(
+    FunctionParameters.create(
+        calculateHealthChange(
+            healthPointsDefaults.redAverage,
+            healthPointsDefaults.blackMax,
+            3
+        ),
+        0,
+        0,
+        0
+    ),
+    [
+        {
+            matchingHealthStateId: noChangesState.id,
+            maximumHealth: healthPointsDefaults.blackMax,
+        },
+    ]
+);
+
+const redIntoTransportFor2PhasesState = PatientHealthState.create(
+    FunctionParameters.create(0, 0, 0, 0),
+    [
+        {
+            matchingHealthStateId: redInstantTransportState.id,
+            earliestTime: 2 * phaseTime,
+        },
+    ]
+);
+
+const redUntilPhase2State = PatientHealthState.create(
+    FunctionParameters.create(
+        calculateHealthChange(
+            healthPointsDefaults.redAverage,
+            healthPointsDefaults.blackMax,
+            2
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.redAverage,
+            healthPointsDefaults.blackMax,
+            2
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.redAverage,
+            healthPointsDefaults.blackMax,
+            2
+        ),
+        0
+    ),
+    [
+        {
+            matchingHealthStateId: noChangesState.id,
+            maximumHealth: healthPointsDefaults.blackMax,
+        },
+        {
+            matchingHealthStateId: redIntoTransportFor2PhasesState.id,
+            earliestTime: 3 * phaseTime,
+        },
+    ]
+);
+
+const prioRedUntilPhase2State = PatientHealthState.create(
+    FunctionParameters.create(
+        calculateHealthChange(
+            healthPointsDefaults.redAverage,
+            healthPointsDefaults.blackMax,
+            2
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.redAverage,
+            healthPointsDefaults.blackMax,
+            2
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.redAverage,
+            healthPointsDefaults.blackMax,
+            2
+        ),
+        0
+    ),
+    [
+        {
+            matchingHealthStateId: noChangesState.id,
+            maximumHealth: healthPointsDefaults.blackMax,
+        },
+        {
+            matchingHealthStateId: redInstantTransportState.id,
+            earliestTime: 3 * phaseTime,
+        },
+    ]
+);
+
+export const defaultPatientCategories: readonly PatientCategory[] = [
+    // XAXAXA Patients
+    PatientCategory.create('XAXAXA', defaultMaleImage, [
```

Comment:
Just as a reminder: In the long run, rename this image and remove the other one. This name gets confusing a few lines below.